core/api/v1/keytransparency.proto

```diff
@@ -65,7 +66,13 @@ message Entry {
   // modifying creating a hash chain of all mutations. The hash used is
   // CommonJSON in "github.com/benlaurie/objecthash/go/objecthash".
   bytes previous = 8;
+}
```

Comment:
Can you mark tag 2 as reserved? (and maybe 1, 4, 5 as well, since it looks like those are unused)